packages/nodes-base/nodes/Form/common.descriptions.ts

```diff
@@ -314,3 +314,12 @@ export const respondWithOptions: INodeProperties = {
 		},
 	],
 };
+
+export const appendAttribution: INodeProperties = {
+	// eslint-disable-next-line n8n-nodes-base/node-param-display-name-miscased
+	displayName: 'Append n8n Attribution',
+	name: 'appendAttribution',
+	type: 'boolean',
+	default: true,
+	description: 'Whether to include the link “Form automated with n8n” at the bottom of the form',
```

Comment:
besides the `description` all instances of this property in the codebase are identical. Should we extract all of these into a common place instead to make sure that no one accidentally creates a new property name in the future?